sbatch/new_repo_train.sh

```diff
@@ -1,17 +1,18 @@
 #!/bin/bash
 #
 #SBATCH --job-name=fermi
-#SBATCH --output=/home/aaronz/2023-Autumn-Clinic-Fermi-CaloDiffusionPaper/logs/%j.%N.stdout
-#SBATCH --error=/home/aaronz/2023-Autumn-Clinic-Fermi-CaloDiffusionPaper/logs/%j.%N.stderr
-#SBATCH --chdir=/home/aaronz/2023-Autumn-Clinic-Fermi-CaloDiffusionPaper/scripts
+#SBATCH --output=/home/isaacharlem/2023-Autumn-Clinic-Fermi-CaloDiffusionPaper/logs/%j.%N.stdout
+#SBATCH --error=/home/isaacharlem/2023-Autumn-Clinic-Fermi-CaloDiffusionPaper/logs/%j.%N.stderr
+#SBATCH --chdir=/home/isaacharlem/2023-Autumn-Clinic-Fermi-CaloDiffusionPaper/scripts
```

Comment:
Could you change this to the flexible ${USER} that you used in line 11, and double-check that that works?